dowhy/causal_graph.py

```diff
@@ -449,11 +450,11 @@ def get_all_directed_paths(self, nodes1, nodes2):
     def has_directed_path(self, nodes1, nodes2):
```

Comment:
This method is not used anywhere in the code-base, but I think it would make sense for this method to just invoke the has_directed_path in the dowhy.graph file, instead of maintaining the same code in two places